hwilib/commands.py

```diff
@@ -210,9 +209,12 @@ def process_commands(args):
     getkeypol_parser = subparsers.add_parser('getkeypool', help='Get JSON array of keys that can be imported to Bitcoin Core with importmulti')
     getkeypol_parser.add_argument('--internal', action='store_true', help='Indicates that the keys are change keys')
     getkeypol_parser.add_argument('--keypool', action='store_true', help='Indicates that the keys are to be imported to the keypool')
-    getkeypol_parser.add_argument('path_base', help='The prefix of the derivation path')
-    getkeypol_parser.add_argument('start', type=int, help='The index to start at. The first key will be <path_base>/<start>')
-    getkeypol_parser.add_argument('end', type=int, help='The index to end at. The last key will be <path_base>/<end>')
+    getkeypol_parser.add_argument('path_base', help='The prefix of the derivation path, e.g. m/84h/0h/0h for account 0 native SegWit')
+    getkeypol_parser.add_argument('path_suffix', help='The suffix of the derivation path, e.g. 0/* for receive and 1/* for change')
```

Comment:
What is the point of splitting the `path_base` into both `path_base` and `path_suffix`? The `path_base` can be used for both.

Reply:
Ah, I see. Then in that case, this would be fine.